catalyst-gateway/tests/api_tests/utils/auth_token.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime, timezone
+import base64
+import pytest
+from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PrivateKey, Ed25519PublicKey
+
+@pytest.fixture
+def rbac_auth_token() -> str:
```

Comment:
I think it would be better to not return a factory method (look at `comment_doc_factory` fixture or `proposal_doc_factory`), because inside the test itself in some cases you would want to generate more than one auth token and probably control the time when it will be generated